src/TodoList.jsx

```diff
@@ -36,11 +36,10 @@ function TodoList() {
 
   return (
     <div className='card'>
-      
+
       <h1 className='card-title'>Todo List</h1>
-      
+
       <div className='custom_input'>
-        <svg xmlns="http://www.w3.org/2000/svg" className="svg_icon bi-search" viewBox="0 0 16 16"><path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"></path></svg>
```

Comment:
**UX suggestion:** this was rendering a 🔎 icon... 

so i thought you did implement a search functionality by name? the expected feature from a 🔎 and a text input on top a list is to... filter the list!